frontend/public/components/project-overview.jsx

```diff
@@ -0,0 +1,341 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { Button, EmptyState, Icon, Toolbar } from 'patternfly-react';
+import { Helmet } from 'react-helmet';
+import { StartGuide } from './start-guide';
+import { FLAGS, connectToFlags, flagPending } from '../features';
+import { Firehose, NavTitle, ResourceName, SectionHeading } from './utils';
+import { DeploymentConfigOverviewList } from './deployment-config';
+
+const LIST_VIEW = 'list';
+const GRAPH_VIEW = 'graph';
+
+const ProjectOverviewToolbar = ({activeView, setView}) => {
+  return <Toolbar>
```

Comment:
@sg00dwin ping!
I know you are consolidating a handful of toolbars throughout the console, would be good to take a quick look at this for future consistency.